src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -1,659 +1,91 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
I'm no longer able to set the WebHost as the startup project and F5 - I get the following failure:

```
System.IO.FileLoadException occurred
  HResult=0x80131040
  Message=Could not load file or assembly 'System.Diagnostics.DiagnosticSource, Version=4.0.2.1, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
  Source=<Cannot evaluate the exception source>
  StackTrace:
   at Microsoft.AspNetCore.Hosting.WebHostBuilder.BuildCommonServices(AggregateException& hostingStartupErrors)
   at Microsoft.AspNetCore.Hosting.WebHostBuilder.Build()
   at WebJobs.Script.WebHost.Core.Program.BuildWebHost(String[] args) in C:\code\Git\Azure\azure-webjobs-sdk-script\src\WebJobs.Script.WebHost\Program.cs:line 22
   at WebJobs.Script.WebHost.Core.Program.Main(String[] args) in C:\code\Git\Azure\azure-webjobs-sdk-script\src\WebJobs.Script.WebHost\Program.cs:line 16
```

Reply:
Checked with @mathewc and he was running outdated SDK bits